src/components/HomePage.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import MovieCard from "./MovieCard";
+
+import { Container } from "../styles/Container.styled";
+import GlobalStyles from "../styles/Global.styled";
+import { Cards } from "../styles/Cards.styled"
+import PageNotFound from "./PageNotFound";
+import {apiUrl,searchUrl} from "./constants/Global"
+
+
+function HomePage(){
+  const [movies, setMovies] = useState([]);
+  const [searchTerm, setSearchTerm] = useState("");
+
+  useEffect(() => {
+    getMovies(apiUrl)
+  },
+    [])
+
+  async function getMovies(url) {
```

Comment:
declare function before calling the function. Please use array function syntax instead of function keyword.